dali/kernels/signal/fft/fft_cpu_impl_utils.h

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_KERNELS_SIGNAL_FFT_FFT_CPU_IMPL_UTILS_H_
+#define DALI_KERNELS_SIGNAL_FFT_FFT_CPU_IMPL_UTILS_H_
+
+#include <utility>
+#include <vector>
+
+namespace dali {
+namespace kernels {
+namespace signal {
+namespace fft {
+namespace impl {
+
+inline int64_t next_pow2(int64_t n) {
+  int64_t pow2 = 2;
+  while (n > pow2) {
+    pow2 *= 2;
+  }
+  return pow2;
+}
+
+inline bool is_pow2(int64_t n) {
+  return (n & (n-1)) == 0;
+}
+
+inline bool can_use_real_impl(int64_t n) {
```

Comment:
This is only true for FFTS -> should be in ffts impl?

Reply:
done